TimeFormat.java

```diff
@@ -12,6 +12,20 @@ public static void main(String[] args) {
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
 		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+		
+		int newHour = hours % 12;
+		if(hours == 12) 
+		{
+		    newHour = 12;
+		}
+		System.out.print(newHour + ":" + args[0].charAt(3) + args[0].charAt(4) +" ");

```

Comment:
Directly referencing the string characters at args[0] is efficient, but less readable